assets/js/modules/thank-with-google/components/setup/SetupMain.js

```diff
@@ -101,8 +101,8 @@ export default function SetupMain( { finishSetup } ) {
 							<div className="googlesitekit-setup-module__heading">
 								<div className="googlesitekit-setup-module__logo">
 									<ThankWithGoogleIcon
-										width="33"
-										height="33"
+										width="40"
+										height="39"
```

Comment:
Comparing with others, it looks like 33x33 is correct. Why would we make it larger?

![image](https://user-images.githubusercontent.com/1621608/181077950-c1a2e904-88a8-49d3-ab6f-05d23aaec28a.png)
![image](https://user-images.githubusercontent.com/1621608/181077966-d4455f34-b088-4f8e-af4e-e46adad7e526.png)

Currently on develop:
![image](https://user-images.githubusercontent.com/1621608/181078013-eff58853-14bc-456c-abd4-946fce2e70a8.png)